.github/workflows/pre-release.yml

```diff
@@ -1,14 +1,19 @@
 name: Release 2 - Pre-release testing
-run-name: Pre-release Testing of ${{ github.event.workflow_run.name }}
+run-name: Publish Release ${{ inputs.version }}
```

Comment:
I'm confused by this. We're not publishing a release, aren't we?
This workflow is supposed to download the official artifact, test it and then push it to our staging Dockerhub repo.

Reply:
good catch, this was a copy-pasta error when I changed the variable